test/gurka/test_closure_penalty.cc

```diff
@@ -172,9 +172,6 @@ TEST_P(ClosurePenalty, AvoidClosure) {
             .str();
 
     std::vector<std::string> expected_path = {"QT", "TU", "RU", "OR", "LO", "IL", "FI", "CF"};
-    if (costing == "motor_scooter") {
```

Comment:
@mandeepsandhu I've seen a comment about this change in your PR https://github.com/valhalla/valhalla/pull/2964#issuecomment-806217728
I guess this is insignificant(as you noted)